app/controllers/admin/documentations_controller.rb

```diff
@@ -49,4 +50,8 @@ def documentation_params
   def assign_documentation
     @documentation = Documentation.find(params[:id])
   end
+
+  def pluck_cover_path_documentation
+    @documentations = Documentation.pluck(:cover_path)
```

Comment:
Nice!
To be more precise - and not confusing, especially when into the view - I recommend you to rename this `@documentations_cover_path`